backend/html/emailTemplate.html

```diff
@@ -4,47 +4,36 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Lynx Locks Account Key Registration</title>
-    <style>
-        body {
-            font-family: Arial, sans-serif;
-            line-height: 1.6;
-            margin: 0;
-            padding: 0;
-            background-color: #f4f4f4;
-            display: flex;
-            justify-content: center;
-            align-items: center;
-            height: 100vh;
-        }
-        .container {
-            max-width: 600px;
-            background-color: #fff;
-            padding: 20px;
-            border-radius: 8px;
-            box-shadow: 0 0 10px rgba(0,0,0,0.1);
-            text-align: center;
-        }
-        .button {
-            display: inline-block;
-            background-color: #007bff;
-            color: #fff;
-            text-decoration: none;
-            padding: 10px 20px;
-            border-radius: 5px;
-            font-weight: bold;
-            transition: color 0.3s;
-        }
-        .button:hover {
-            color: #800080;
-        }
-    </style>
 </head>
-<body>
-<div class="container">
-    <p>A request was recently made to register a new key on your Lynx Locks account.</p>
-    <p>Please use the below link to add your passkey/yubikey:</p>
-    <a href={{.Link}} class="button">Add Your Key</a>
-    <p>If you believe this to be an error, contact your administrator.</p>
-</div>
+<body style="font-family: Arial, sans-serif; line-height: 1.6; margin: 0; padding: 0; background-color: #ffffff; color: #000000;">
+<style>
+    @media only screen and (max-width: 600px) {
+        .desktop-only {
+            border: none !important;
+        }
+    }
+</style>
+
+<!--[if (gte mso 9)|(IE)]>
+<table role="presentation" border="0" cellpadding="0" cellspacing="0" width="600" align="center">
+    <tr>
+        <td>
+<![endif]-->
+<table class="desktop-only" style="width: 100%; max-width: 600px; margin: 0 auto; background-color: #ffffff; padding: 20px; border-radius: 8px; border: 2px solid #000000; border-collapse: collapse;">
+    <tr>
+        <td style="padding-left: 30px; margin-left: 5px;">
+            <h1 style="font-weight: bold; color: #000000;">Key registration request for Lynx Locks</h1>
+            <p style="color: #000000; margin-bottom: 20px;">Hi, {{.Name}}, please use the below link to add a key to your Lynx Locks account.</p>
+            <p style="color: #000000; margin-bottom: 20px;">Be sure to use it on a device capable of registering passkeys or a YubiKey!</p>
+            <a href="{{.Link}}" style="display: inline-block; background-color: rgb(242, 155, 4); color: #000000; text-decoration: none; padding: 15px 25px; border-radius: 8px; font-weight: bold; font-size: 16px;">Add Your Key</a>
```

Comment:
Does <button> not work on this?

Reply:
Why is the orange different on light / dark? Also why does the first one have no padding?

Reply:
Also why did you change all the styling to be inline?

Reply:
idk what ur first comment is, the 2nd is its literally just outlook being weird, it shouldn't be different,
the styling is inline as sometimes outlook has a hard time rendering complex styling hence this was an attempt, and then further changes were done from there so it was left

Reply:
the issue with the orange is probably (definitely) cuz outlook darkmode isn't real darkmode they just take your colours and invert them

Reply:
also idk why one u said has no padding, i think all my images do?

Reply:
![Screenshot 2024-03-13 at 10 58 45 PM](https://github.com/lynx-locks/lynx-web/assets/55325093/c13fd987-943b-4290-a5fc-eb03bb3a1d38)

Reply:
ah i see the button, yea another outlook special if you use new outlook everything is fine
![image](https://github.com/lynx-locks/lynx-web/assets/57304476/23d58fe6-8268-4977-907c-890e753320d9)
should've added that in my description mb

Reply:
Why do you have all the styling inline?

Reply:
It was supposed to be easier for emails to interpret, (not sure how much affect it actually ended up happening) but that is why it was transitioned and then why it was left that way as per wat i said earlier